madara/crates/client/db/src/storage_updates.rs

```diff
@@ -88,6 +88,10 @@ impl MadaraBackend {
         Ok(())
     }
 
+    pub fn revert_to(&self, revert_to: u64) -> Result<(), MadaraStorageError> {
+        self.block_db_revert(revert_to)
```

Comment:
reverting block_db is not sufficient, but I think you're aware of it as we talked about it before
(sorry if this pr was supposed to be a draft)

Reply:
Can you elaborate? We also need to revert the bonsai tries, but that's done elsewhere. Is that what you mean?

Reply:
[d8b045f](https://github.com/madara-alliance/madara/pull/296/commits/d8b045f221e069a29b06c2b1ec6c7efa2a5fa364) adds support for removing declared class info and rolls back contract state. It's not optimized but should suffice for small reorgs, or at least for early testing :)